tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -2984,4 +2984,58 @@ public function test_doctype_doc_name() {
 		$this->assertNull( $doctype->public_identifier );
 		$this->assertNull( $doctype->system_identifier );
 	}
+
+	/**
+	 * @ticket 62522
+	 *
+	 * @dataProvider data_alphabet_by_characters_lowercase
+	 */
+	public function test_recognizes_lowercase_tag_name( string $char ) {
+		$html      = " <{$char}> </{$char}>";
```

Comment:
Maybe we should indicate that blank space is on purpose? I'm referring the one after the "

Reply:
Sure, added.